// async function addAsync(a, b) {
//     return a + b;
// }

// async function performAddition() {
//     try {
//         let result = await addAsync(5, 3);
//         console.log("Result of addition:", result);
//     } catch (error) {
//         console.error("Error:", error);
//     }
// }

// performAddition();
